resources/public/api/conf/common/versioning.md

```diff
@@ -0,0 +1 @@
+When [backwards-incompatible changes](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/documentation/how-to-integrate.html#breaking-changes) are made to this API, we release a new version. Read the [reference guide](https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#versioning) for more information about API versioning. For updates about all our APIs, refer to the [API changelog](https://github.com/hmrc/income-tax-mtd-changelog).
```

Comment:
Environment specific url can be used: `[reference guide](/api-documentation/docs/reference-guide#versioning)`